packages/fuel-indexer-api-server/src/bin/fuel-indexer-api-server.rs

```diff
@@ -27,7 +27,7 @@ pub async fn main() -> Result<()> {
 
     info!("Configuration: {:?}", config);
 
-    GraphQlApi::run(config.clone(), None).await;
+    GraphQlApi::run(config.clone(), None, None).await;
```

Comment:
Mentioned above ☝️ but the `GraphQlApi` w/o a DB connection can't really do anything so we can pass one here.